reana-ui/src/reducers.js

```diff
@@ -30,6 +35,8 @@ import {
 } from "./actions";
 import { USER_ERROR } from "./errors";
 
+const errorInitialState = null;
```

Comment:
One last comment which doesn't have to be addressed in this PR. Should we make the error field a list? Right now if I have two errors in a row in a short time, the second error "overwrites" the first and I can only see the latter.